### ISSUE TITLE
Update jaxws-ri to version which is supported by maven 3.8

### ISSUE THREAD

**Issue by WouterLangerak** [spring-projects/spring-boot#30216]
Building a spring boot application with `jaxws-ri` with maven 3.8 fails. The failure is caused by the usage of an http repository, which is blocked by maven since version 3.8.1.

**Usage**
```xml
<dependency>
  <groupId>com.sun.xml.ws</groupId>
  <artifactId>jaxws-ri</artifactId>
  <version>${javax-jaxws.version}</version>
  <type>pom</type>
</dependency>
```

**Fix**
Overriding this version to `2.3.5` fixes the build

**Comment by bclozel** [comment]
To me Spring Boot only declared dependency management for `javax.xml.ws:jaxws-api`, but not the -ri dependency. Where is this dependency coming from? Can you share the dependency tree or insights for your application?

Am I missing something?

**Comment by WouterLangerak** [comment]
You are right, `jaxws-ri` is not in the dependency management. So using the `javax-jaxws.version` from spring boot's properties for the  `jaxws-ri` dependency is not correct.
I'll fix it in our project and close this issue.

Thank you for your quick response.
